src/btree/bt_compact.c

```diff
@@ -314,7 +314,7 @@ __wt_compact(WT_SESSION_IMPL *session)
     WT_DECL_RET;
     WT_REF *ref;
     u_int i;
-    bool first, skip;
+    bool first, skip, didwork;
```

Comment:
Please declare in alphabetical order and I also suggest a different name to know what it refers to:

```suggestion
    bool eviction_happened, first, skip;
```